docs/cql/ddl.rst

```diff
@@ -109,15 +109,16 @@ For example:
 
 The supported ``options`` are:
 
-=================== ========== =========== ========= ===================================================================
-name                 kind       mandatory   default   description
-=================== ========== =========== ========= ===================================================================
-``replication``      *map*      no                    The replication strategy and options to use for the keyspace (see
-                                                      details below).
-``durable_writes``   *simple*   no          true      Whether to use the commit log for updates on this keyspace
-                                                      (disable this option at your own risk!).
-``tablets``          *map*      no                    Enables or disables tablets for the keyspace (see :ref:`tablets <tablets>`)
-=================== ========== =========== ========= ===================================================================
+=================== ========== =========== =========== ===================================================================
+name                 kind       mandatory    default    description
+=================== ========== =========== =========== ===================================================================
+``replication``      *map*      no                      The replication strategy and options to use for the keyspace (see
+                                                        details below).
+``durable_writes``   *simple*   no           true       Whether to use the commit log for updates on this keyspace
+                                                        (disable this option at your own risk!).
+``tablets``          *map*      no                      Enables or disables tablets for the keyspace (see :ref:`tablets <tablets>`)
+``consistency``      *simple*   no          `eventual`  Configures consistency mode for the keyspace (see :ref:`consistency <consistency-option>`).
```

Comment:
Should we make it a map to extend it later?

I guess we can add a map option if we want.